build-tools/packages/build-cli/src/config.ts

```diff
@@ -63,6 +63,22 @@ export interface FlubConfig {
 	 * Configuration for the `generate:releaseNotes` command.
 	 */
 	releaseNotes?: ReleaseNotesConfig;
+
+	/**
+	 * The multiple of minor versions to use for calculating the next version in the legacy compatibility range.
+	 */
+	releaseReport?: LegacyCompatInterval;
+}
+
+/**
+ * Represents legacy compat interval per release groups.
+ */
+export interface LegacyCompatInterval {
```

Comment:
I think the interface name should be `ReleaseReportConfig`. Legacy compat intervals are settings with in. Update the comments to avoid saying specifically for legacy compat even though that is all it has right now.